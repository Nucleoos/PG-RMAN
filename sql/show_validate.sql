-- test show command
\! rm -rf ${PWD}/results/sample_backup
\! cp -rp data/sample_backup ${PWD}/results/sample_backup
\! pg_rman show -B ${PWD}/results/sample_backup
\! pg_rman validate -B ${PWD}/results/sample_backup 2009-05-31 17:05:53 --debug
\! pg_rman validate -B ${PWD}/results/sample_backup 2009-06-01 17:05:53 --debug
\! pg_rman show -a -B ${PWD}/results/sample_backup
\! pg_rman show 2009-06-01 17:05:53 -B ${PWD}/results/sample_backup
