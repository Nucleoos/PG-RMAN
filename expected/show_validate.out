-- test show command
\! rm -rf ${PWD}/results/sample_backup
\! cp -rp data/sample_backup ${PWD}/results/sample_backup
\! pg_rman show -B ${PWD}/results/sample_backup
============================================================================
Start                Time   Total    Data     WAL     Log  Backup   Status  
============================================================================
2009-06-03 17:05:53    0m    ----    ----    ----    ----    ----   RUNNING
2009-06-01 17:05:53    3m    ----  9223PB    16MB    ----   162MB   DONE
2009-05-31 17:05:53    3m  1242MB    ----  9223PB    ----   242MB   DONE
\! pg_rman validate -B ${PWD}/results/sample_backup 2009-05-31 17:05:53 --debug
INFO: validate: 2009-05-31 17:05:53 backup and archive log files by CRC
LOG: database files...
LOG: (1/1) PG_VERSION
LOG: archive WAL files...
LOG: backup 2009-05-31 17:05:53 is valid
\! pg_rman validate -B ${PWD}/results/sample_backup 2009-06-01 17:05:53 --debug
INFO: validate: 2009-06-01 17:05:53 backup and archive log files by CRC
LOG: database files...
LOG: (1/1) PG_VERSION
WARNING: CRC of backup file "PG_VERSION" must be 0 but FEF71BC1
LOG: archive WAL files...
WARNING: backup 2009-06-01 17:05:53 is corrupted
\! pg_rman show -a -B ${PWD}/results/sample_backup
============================================================================
Start                Time   Total    Data     WAL     Log  Backup   Status  
============================================================================
2009-06-03 17:05:53    0m    ----    ----    ----    ----    ----   RUNNING
2009-06-02 17:05:03    0m    ----    ----    ----  4335kB   162MB   DELETED
2009-06-01 17:05:53    3m    ----  9223PB    16MB    ----   162MB   CORRUPT
2009-05-31 17:05:53    3m  1242MB    ----  9223PB    ----   242MB   OK
\! pg_rman show 2009-06-01 17:05:53 -B ${PWD}/results/sample_backup
# configuration
BACKUP_MODE=INCREMENTAL
WITH_SERVERLOG=false
COMPRESS_DATA=false
# result
TIMELINEID=1
START_LSN=0/0b40c800
STOP_LSN=0/0b4c8020
START_TIME='2009-06-01 17:05:53'
END_TIME='2009-06-01 17:09:13'
RECOVERY_XID=0
TOTAL_DATA_BYTES=1242102558
READ_DATA_BYTES=9223372036854775807
READ_ARCLOG_BYTES=16777216
WRITE_BYTES=162372983
BLOCK_SIZE=8192
XLOG_BLOCK_SIZE=8192
STATUS=CORRUPT
